gateway/scim_extensions.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024, Cisco Systems, Inc. and/or its affiliates.
+# All rights reserved.
+# See LICENSE file in this distribution.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+
+This module implements SCIM extensions in a generic fashion.
+
+"""
+
+scim_ext_create = []
+scim_ext_read = []
+scim_ext_update = []
+scim_ext_delete = []
```

Comment:
I think this should be pre-populated with all the functions and the individual function should throw an error if `WANT_FDO` or similar is false.

Reply:
If you do that, then you have to edit yet more files to add an extension.